<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorInfoDialog</class>
 <widget class="QDialog" name="SensorInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>History:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="edit_new_value">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Set New Value</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sensor Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="edit_sensor_name">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current Value:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QLabel" name="label_one">
         <property name="text">
          <string>Time On (s):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_off">
         <property name="text">
          <string>Time Off (s):</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Get History</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="cb_values"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_consumption_2">
         <property name="text">
          <string>Total Consumption:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="label_time_off">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>History Values:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Power Consumption:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="label_time_on">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLabel" name="label_consumption">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QComboBox" name="cb_period">
         <item>
          <property name="text">
           <string>6 hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 day</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7 days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 month</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Period:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="cb_bulb_type">
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="edit_current_value">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>New Value:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
